ppr-api/src/ppr_api/models/draft.py

```diff
@@ -58,6 +58,7 @@ class DraftTypes(Enum):
     draft = db.Column('draft', db.JSON, nullable=False)
     registration_number = db.Column('registration_number', db.String(10), nullable=True)
     update_ts = db.Column('update_ts', db.DateTime, nullable=True)
+    user_id = db.Column('user_id', db.String(1000), nullable=True)
```

Comment:
should this be a foreign key to make it easier to link?

Reply:
I considered it, but the direct api consumers may submit requests with no users record, throwing a foreign constraint violation. With a constraint, we need to check/insert every time on high volume get drafts/get financing statements/get registrations requests. So I left the constraint out and use subqueries which allow for no users record.